test/parallel/test-fs-read-stream-double-close.js

```diff
@@ -3,7 +3,18 @@
 const common = require('../common');
 const fs = require('fs');
 
-const s = fs.createReadStream(__filename);
+{
+  const s = fs.createReadStream(__filename);
 
-s.close(common.mustCall());
-s.close(common.mustCall());
+  s.close(common.mustCall());
+  s.close(common.mustCall());
+}
+
+{
+  const s = fs.createReadStream(__filename);
+
+  // this is a private API, but it is worth
```

Comment:
Minor nit: `This is a private API, but it is worth testing. close() calls this.` fits on one line.